rosalind_lcsm.py

```diff
@@ -0,0 +1,24 @@
+def is_in_each(sub, dna_strings):
+    for dna_string in dna_strings:
+        if not sub in dna_string:
+            return False
+    return True
```

Comment:
Yup. I had that first, but thought there was an issue with it, but the issue was my `while` loop.